pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/dictionary/FloatMutableDictionary.java

```diff
@@ -155,20 +177,70 @@ public boolean inRange(String lower, String upper, int indexOfValueToCompare, bo
 
   @Override
   public String getStringValue(int dictionaryId) {
-    return (getRawValueFromBiMap(dictionaryId)).toString();
-  }
-
-  private float getFloat(int dictionaryId) {
-    return (Float) dictionaryIdBiMap.get(dictionaryId);
+    return ((Float) _dictionaryIdBiMap.getValue(dictionaryId)).toString();
   }
 
   @Override
   public Object getMinVal() {
-    return min;
+    return _min;
   }
 
   @Override
   public Object getMaxVal() {
-    return max;
+    return _max;
+  }
+
+  @Override
+  public void print() {
```

Comment:
There are existing callers from print (which may be for debug mode). I'd skip removing that in this PR.